ROADMAP.md

```diff
@@ -0,0 +1,45 @@
+# Tekton Hub 2021 Roadmap
+
+## Roadmap
+
+### Multi Catalog Support
+
+ - User should be able to provide multi catalogs in config
+ - API's should be able to act for multiple catalogs
+ - UI should show the resources from all catalogs mentioned and also able to filter based on catalog
+ - CLI should be able to work with multiple catalogs
+
+### Add support for Pipeline Kind
+
+ - Hub should the Pipelines Kind resources
+ - APIs created to get Pipeline Kind data
+ - CLI should be able to install, upgrade etc for Pipelines
+ - Catlin should be able to validate based on Pipeline Structure Proposal
+
+### Handle Deprecated Task in UI
+
+ - UI should have a way to identify deprecated task
+ - API should return whether the resource is deprecated or not
+
+### CLI acts based on Minimum Pipeline Version of resource
+
+ - CLI commands like install, upgrade, etc act based on Pipeline version installed on cluster
+ - It should warn user in case of mismatch with supported Pipeline version of resource
+
+### Automate Hub Deployment
+
+ - Hub deployment should be automated through script.
+
+### Catlin Integration
+
+ - Catlin should be integrated with catalog CI to validate resource according to TEP
+ - Catlin should support both Task and Pipeline TEP proposal
```

Comment:
This isn't *really* related to Hub, but our catalog, am I right ?

Reply:
removed from RoadMap